src/languages/handlebars.js

```diff
@@ -3,8 +3,8 @@ Language: Handlebars
 Requires: xml.js
 Author: Robin Ward <robin.ward@gmail.com>
 Description: Matcher for Handlebars as well as EmberJS additions.
-Website: https://handlebarsjs.com
-Category: template
+Website: https://handlebarsjs.com, https://emberjs.com
+Category: common, template
```

Comment:
Please revert, this is not your decision to make - which languages are in our `common` set.  If you just want to add handlebars to the build your an do so on the console:

```
./tools/build -t node :common handlebars
```

Etc.

---

Also what is embers relationship to Handlebars?

Reply:
> Also what is embers relationship to Handlebars?

ember has similar syntax, but has built a bunch of stuff on top, as a result, the handlebars syntax in highlight.js ~7 years out of date :(
(but that's why I'm making a PR!) :D

Reply:
> as a result, the handlebars syntax in highlight.js ~7 years out of date

I'm not sure that's a correct statement [in the general sense]. I know @nknapp put a lot of work into it just last year to improve it.

https://handlebarsjs.com  

AFAIK Handlebar exists as it's own independent project and there are surely those using it outside of Ember - _like our very own project, actually_.   So the question then becomes whether this is appropriate to be added to Handlebars at all _or whether a separate Ember grammar may be needed_.  I'm curious if nknapp has any thoughts on that. 

In any case I can certainly see how this would be helpful for Ember users, but if we decide it's not appropriate as part of Handlebars proper then it would need to be distributed as a 3rd party Ember grammar - rather than mixed with our existing support.

Reply:
Yeah that's totally fair! Most editors have an ember-variant to use for hbs highlighting